### PR TITLE
Update site for 2024/25 course 

### PR DESCRIPTION
**Pull request by NovIrina**

### CODE REVIEW

File: source/docs/labs_2024/general_info.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+-  `Тугарёв Артём
+   Михайлович <https://www.hse.ru/org/persons/224103384>`__ -
+   преподаватель практики
+-  `Кащихин Андрей Николаевич <https://github.com/WhiteJaeger>`__ -
+   приглашённый эксперт
+-  `Жариков Егор Игоревич <https://t.me/godb0i>`__ - ассистент
+-  `Новикова Ирина Алексеевна <https://t.me/iriinnnaaaaa>`__ - ассистент
+-  `Блюдова Василиса Михайловна <https://t.me/Vasilisa282>`__ -
+   ассистент
+-  `Зайцева Вита Вячеславовна <https://t.me/v_ttec>`__ - ассистент
```
Comment (demid5111) [inline source/docs/labs_2024/general_info.rst:18]: she is also an expert

File: source/docs/labs_2024/general_info.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+-  `Демидовский Александр
+   Владимирович <https://www.hse.ru/staff/demidovs>`__ - лектор
+-  `Тугарёв Артём
+   Михайлович <https://www.hse.ru/org/persons/224103384>`__ -
+   преподаватель практики
+-  `Кащихин Андрей Николаевич <https://github.com/WhiteJaeger>`__ -
+   приглашённый эксперт
+-  `Жариков Егор Игоревич <https://t.me/godb0i>`__ - ассистент
+-  `Новикова Ирина Алексеевна <https://t.me/iriinnnaaaaa>`__ - ассистент
+-  `Блюдова Василиса Михайловна <https://t.me/Vasilisa282>`__ -
```
Comment (demid5111) [inline source/docs/labs_2024/general_info.rst:16]: practice teacher

File: source/docs/labs_2024/general_info.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+-  `Демидовский Александр
+   Владимирович <https://www.hse.ru/staff/demidovs>`__ - лектор
+-  `Тугарёв Артём
+   Михайлович <https://www.hse.ru/org/persons/224103384>`__ -
+   преподаватель практики
+-  `Кащихин Андрей Николаевич <https://github.com/WhiteJaeger>`__ -
+   приглашённый эксперт
+-  `Жариков Егор Игоревич <https://t.me/godb0i>`__ - ассистент
+-  `Новикова Ирина Алексеевна <https://t.me/iriinnnaaaaa>`__ - ассистент
```
Comment (demid5111) [inline source/docs/labs_2024/general_info.rst:15]: chief assistant

File: source/docs/labs_2024/labs/lab_1_classify_by_unigrams/lab_1.rst [hunk 4]
```diff
@@ -0,0 +1,420 @@
+Лабораторная работа №1. Определение языка текста на основе частотного словаря
+=============================================================================
```
[the remark this answers is not in the document]
Comment (vitazaytseva) [reply source/docs/labs_2024/labs/lab_1_classify_by_unigrams/lab_1.rst]: done

File: source/docs/labs_2024/labs/lab_1_classify_by_unigrams/lab_1_classify_by_unigrams.api.rst [hunk 5]
```diff
@@ -0,0 +1,13 @@
+lab\_1\_classify\_by\_unigrams package
```
[the remark this answers is not in the document]
Comment (vitazaytseva) [reply source/docs/labs_2024/labs/lab_1_classify_by_unigrams/lab_1_classify_by_unigrams.api.rst]: done

File: source/docs/labs_2024/general_info.rst (hunk 1 above)
Comment (vitazaytseva) [reply source/docs/labs_2024/general_info.rst]: done

File: source/docs/labs_2024/general_info.rst (hunk 3 above)
Comment (vitazaytseva) [reply source/docs/labs_2024/general_info.rst]: done

File: source/docs/labs_2024/labs/lab_1_classify_by_unigrams/lab_1.rst [hunk 6]
```diff
@@ -0,0 +1,420 @@
+Лабораторная работа №1. Определение языка текста на основе частотного словаря
```
[the remark this answers is not in the document]
Comment (vitazaytseva) [reply source/docs/labs_2024/labs/lab_1_classify_by_unigrams/lab_1.rst]: Is this a relevant cooment? Now there is a full description in the official repo

**Pull request merged**